broker-plugin/broker-cli/src/main/java/io/enmasse/broker/cli/Cli.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2016-2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.broker.cli;
+
+
+import io.enmasse.amqp.Artemis;
+import io.enmasse.amqp.SyncRequestClient;
+import io.vertx.core.Vertx;
+import io.vertx.core.cli.CLI;
+import io.vertx.core.cli.CommandLine;
+import io.vertx.core.cli.Option;
+import io.vertx.core.cli.impl.DefaultCLI;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.util.Arrays;
+import java.util.concurrent.TimeoutException;
+
+public class Cli implements Closeable {
+
+    private static final Logger log = LoggerFactory.getLogger(Cli.class.getName());
```

Comment:
When using classnames for the logging hierarchy, this way is preferred.  The user can then precisely control the log statements emitted from different subsystems but adjusting the level associated with a whole branch,

private static final Logger log = LoggerFactory.getLogger(Cli.class);